File: CCategoria/CCategoria/Categoria.cs
using System;
namespace CCategoria {
    public class Categoria {
        public Categoria() {
        }
        private long id;
        private string nombre = "";

        public long Id {
            get { return id; }
            set { id = value; }
        }

        public string Nombre {
            get { return nombre; }
            set { nombre = value; }
        }
    }
}

File: GHibernate/src/main/java/serpis/ad/Categoria.java
package serpis.ad;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity

public class Categoria {
	@Id
	@GeneratedValue(strategy=GenerationType.IDENTITY)
	private long id;
	private String nombre;
	
	public String getNombre() {
		return nombre;
	}
	
	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
	
	public long getId() {
		return id;
	}

	public void setId(long id) {
		this.id = id;
	}

	@Override
	public String toString() {
		return String.format("[%s] %s", id, nombre);
	}
}
